runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/resources/ClasspathScanningResourcesDetectorTest.java

```diff
@@ -120,7 +120,7 @@ public void shouldDetectClassPathResourceFromJavaClassPathEnvVariable() throws I
 
   @Test
   public void shouldNotDetectClassPathResourceThatIsNotAFile() throws Exception {
-    String url = "http://www.google.com/all-the-secrets.jar";
+    String url = "http://www.example.com/all-the-secrets.jar";
```

Comment:
Why did we need this change? Was the test failing without the change or was it just to make it more generic?

Reply:
No failures -- just to make it more generic.

Not hard to think that bots / crawler can try to follow that link.

http://www.example.com/ is an IANA domain serves this purpose:

![image](https://github.com/apache/beam/assets/3207647/76cf0221-d598-42fb-bda4-ecff950cf876)

Reply:
Cool - that makes sense, thanks. Just wanted to make sure we weren't changing for a failing test.